node/src/main/scala/com/wavesplatform/utils/UtilApp.scala

```diff
@@ -38,11 +38,13 @@ object UtilApp {
   case class HashOptions(mode: String = "fast")
   case class SignTxOptions(signerAddress: String = "")
 
-  case class Command(mode: Command.Mode = Command.CompileScript,
+  case class Command(mode: Command.Mode = null,
+                     configFile: Option[String] = None,
                      inputData: Option[String] = None,
                      inputFile: Option[String] = None,
```

Comment:
One can specify _either_ literal input string or an input file name, so `Either[String, String]` would be more appropriate.